get.sh

```diff
@@ -156,6 +162,11 @@ getBinaryOpenjdk()
 
 getTestKitGenAndFunctionalTestMaterial()
 {
+	if [[ "$PLATFORM" == "s390x_zos" ]];
+	then
+		OPENJ9_REPO="git://github.com/eclipse/openj9.git"
```

Comment:
OPENJ9_REPO is a parameter, we do not need to hardcode here.

Reply:
We can pass in as parameter from Jenkins file. That's how we do personal build.

Reply:
OK that makes sense for personal builds. From where will the value of OPENJ9_REPO be passed for regular non-personal builds?

Reply:
You can create parameter in Jenkins file configuration and pass in value for OPENJ9_REPO
https://github.com/AdoptOpenJDK/openjdk-tests/blob/5416139f7a6c673e5ceea35b5153aa3e06feed93/buildenv/jenkins/JenkinsfileBase#L122

Reply:
Okay. Makes sense. But why then are we having a default value for this parameter in get.sh for regular case? Why aren't we passing that in via Jenkins file configuration too?

Reply:
We need a default value in case this value is not provided.

Reply:
So can we consider my change to be the default value for the case of z/OS?

Reply:
If we make zos value default, then we have to change all job configuration to pass in https value for OPENJ9_REPO for all other platforms. It does not make sense to me.

Reply:
Not asking to make the z/OS value default. I meant, the way the value is in this PR, can we leave it as is, instead of moving it to a data file (with the check for zos around it), as the "default" value when we are on z/OS.

Reply:
OK. Removed.